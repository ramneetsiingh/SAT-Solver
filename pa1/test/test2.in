1*0+-1;
1; V:1,V:0
