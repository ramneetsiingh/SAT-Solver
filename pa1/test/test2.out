0
Error: contradicting assignment
